assets/scss/_main-container.scss

```diff
@@ -22,13 +22,11 @@
 
 .td-main {
     flex-grow: 1;
+}
 
-    main {
-        @include media-breakpoint-up(md) {
-            padding-top: 5.5rem;
-        }
-
-
-        padding-bottom: 2rem;
-    }
+.td-404 main,
+.td-main main {
+  padding-top: 1.5rem;
+  padding-bottom: 2rem;
+  @include media-breakpoint-up(md) { padding-top: 5.5rem; }
 }
```

Comment:
This change is just a _refactoring_ of the selector `.td-main main` so that the style definition can be reused for `.td-404 main`.